gulpfile.js

```diff
@@ -17,6 +17,7 @@ var gulp = require('gulp'),
     minimatch = require('minimatch'),
     copy = require('gulp-copy'),
     concat = require('gulp-concat'),
+    promise = require('promise'),

```

Comment:
I disagree. I get "ReferenceError: promise is not defined" if I don't declare it in gulpfile.js.